ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -178,9 +171,9 @@ public void processResponse(RestRequest restRequest, RestResponseChannel respons
             setHeadResponseHeaders(blobInfo, options, restRequest, responseChannel);
             break;
           case GET:
-            if (!operationOrBlobId.equals(Operations.GET_SIGNED_URL)) {
+            if (!requestPath.matchesOperation(Operations.GET_SIGNED_URL)) {
               responseChannel.setStatus(ResponseStatus.Ok);
-              RestUtils.SubResource subResource = RestUtils.getBlobSubResource(restRequest);
+              RestUtils.SubResource subResource = RestUtils.getRequestPath(restRequest).getSubResource();
```

Comment:
`getRequestPath` is invoked in 3 places in this file. Two specified the package `RestUtils`, one not. Please make this consistent.

Reply:
removed all `RestUtils` qualifiers from this class since we use a static import